Templates/Per Tenant Extension/.github/workflows/CICD.yaml

```diff
@@ -275,12 +251,20 @@ jobs:
           project: ${{ matrix.project }}
           buildMode: ${{ matrix.buildMode }}
           branchName: ${{ github.ref_name }}
+          
+      - name: Calculate Sanitized Project name
```

Comment:
This is a fix for when an AL-Go project is not in the root folder (so `matrix.project` contains slashes).

Think of a better way to handle this.

Reply:
I'll add it to _CalculateArtifactNames_

Reply:
Added to _CalculateArtifactNames_